src/main/java/com/epam/ta/reportportal/entity/enums/ProjectAttributeEnum.java

```diff
@@ -29,7 +29,7 @@
  */
 public enum ProjectAttributeEnum {
 
-	NOTIFICATIONS_ENABLED("notifications.enabled", String.valueOf(true)),
+	NOTIFICATIONS_ENABLED("notifications.enabled", String.valueOf(false)),
```

Comment:
Check if the invited user gets a email notification

Reply:
Notifications enabled flag isn't used in user invitation sending logic, so it's not affected